### DISCUSSION TITLE
[Feature] turn pictures (and save it)

### DISCUSSION THREAD

**Discussion by Werewolfke** [immich-app/immich#7400] [Feature Request]
### The feature

weirdly enough i dont see this request anywhere, if this is a duplicate, im sorry.
But i have a lot of pictures that are turned 90° (my fault).
So it would be nice to have a button to turn the pictures straight up, and not only for one time viewing, but saving that possition so it always stays that way.

thanks

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1695
